package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MPL-2.0",
   "homepage": "https://pihemr.atlassian.net/wiki/spaces/DSS/pages/538116104/PIH+EMR+Referrals+Queue",
   "scripts": {
-    "start": "openmrs develop --backend https://humci.pih-emr.org/ --api-url /mirebalais/",
+    "start": "openmrs develop --backend https://humci.pih-emr.org --api-url /mirebalais/ --spa-path /mirebalais/spa/ --importmap 'https://humci.pih-emr.org/mirebalais/spa/importmap.json'",
```

Comment:
What the reason for hard-coding here against humci?  Is this specifically run for that environment, or is this a convenience for people to develop/test without needing a local instance?  I'd probably rather this point to localhost.